### PR TITLE
feature/ak-68: Added AWS EC2 VPC Endpoint Exposed plugin, test cases

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

**Review by dipsubha06 — APPROVED** [review]

File: plugins/aws/ec2/vpcEndpointExposed.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'VPC Endpoint Exposed',
+    category: 'EC2',
+    description: 'Ensure Amazon VPC endpoints are not exposed to everyone.',
```
Comment (giorod3) [inline plugins/aws/ec2/vpcEndpointExposed.js:7]:
```suggestion
    description: 'Ensure Amazon VPC endpoints are not publicly exposed',
```

File: plugins/aws/ec2/vpcEndpointExposed.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    }
+                }
+
+                if (!publicEndpoint) {
+                    helpers.addResult(results, 0,
+                        `VPC endpoint ${endpoint.VpcEndpointId} is not exposed`,
+                        region, resource);
+                } else {
+                    helpers.addResult(results, 2,
+                        `VPC endpoint ${endpoint.VpcEndpointId} is exposed to everyone`,
```
Comment (giorod3) [inline plugins/aws/ec2/vpcEndpointExposed.js:63]:
```suggestion
                        `VPC endpoint ${endpoint.VpcEndpointId} is publicly exposed`,
```

**Review by giorod3 — CHANGES_REQUESTED** [review]
a couple of syntax issues, one is in the results so I will leave it for you

File: plugins/aws/ec2/vpcEndpointExposed.js (hunk 1 above)
Comment (AkhtarAmir) [reply plugins/aws/ec2/vpcEndpointExposed.js]: Updated

**Review by giorod3 — APPROVED** [review]

**Pull request merged**